### PR TITLE
Remove path local dependency and replace with project version

### PR DESCRIPTION
**Pull request by cdbrkfxrpt**
re-PR of #113

### CODE REVIEW

File: .changelog/_unreleased.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [[entries]]
 id = "c35a9d23-2db1-4dd0-8e04-088931ee0c9c"
-type = "improvement"
+type = "breaking change"
```
Comment (NiklasRosenstein) [inline .changelog/_unreleased.toml:3]: @cdbrkfxrpt How is this a breaking change?

Comment (cdbrkfxrpt) [reply .changelog/_unreleased.toml:3]: this is the entry for the gitignore thing, which we agreed is a breaking change, and i simply adjusted it in this PR. not the cleanest of ways to do this, admittedly, but... well

Comment (NiklasRosenstein) [reply .changelog/_unreleased.toml:3]: Ah okay 😄

Comment (NiklasRosenstein) [reply .changelog/_unreleased.toml:3]: Not sure when this MR get's merged, I'll update that field separately.

Comment (cdbrkfxrpt) [reply .changelog/_unreleased.toml]: merged now

**Pull request merged**